packages/fuels-test-helpers/Cargo.toml

```
@@ -34,3 +34,4 @@ which = { workspace = true, default-features = false }
 default = ["fuels-accounts", "std"]
 std = ["fuels-accounts?/std", "fuels-core/std"]
 fuel-core-lib = ["fuel-core"]
+rocksdb = ["fuel-core", "fuel-core/rocksdb"]
```

You can use a `?` so that rocksdb only takes affect when fuel-core-lib is enabled.

cc @MujkicA 

```suggestion
rocksdb = ["fuel-core?/rocksdb"]
```